driver.go

```diff
@@ -8,16 +8,26 @@ import (
 
 // register driver
 func init() {
-	sql.Register("maxcompute", &Driver{})
+	sql.Register("maxcompute", &ODPSDriver{})
```

Comment:
I'd rather use `MaxcomputeDriver` than `ODPSDriver`.
In fact, we use `odps*` as a name within this repo.

Reply:
Right, maxcompute is the open-source name.